java/org/brotli/wrapper/enc/encoder_jni.cc

```diff
@@ -79,6 +79,10 @@ Java_org_brotli_wrapper_enc_EncoderJNI_nativeCreate(
     if (lgwin >= 0) {
       BrotliEncoderSetParameter(handle->state, BROTLI_PARAM_LGWIN, lgwin);
     }
+    int mode = context[4];
+    if (mode >= 0) {
```

Comment:
`mode > 0` as with lgwin...

Reply:
If `mode` is null as you suggested above then what will be the value here ?

Reply:
Oh, you are right, `0` is valid value. Just like with lgwin we shoud set it to -1 if enum value is null

Reply:
OK. I will change it now!

Reply:
Done!